File: server/model/product.go
package model

type Product struct {
	ID           int     `json:"id"`
	UPC          string  `json:"upc"`
	Name         string  `json:"name"`
	Vendor       string  `json:"vendor"`
	IsActive     bool    `json:"isActive"`
	UnitCost     float64 `json:"unitCost"`
	RetailPrice  float64 `json:"retailPrice"`
	UnitsInStock int     `json:"unitsInStock"`
}

File: client/EZ-Inventory/EZ-Inventory/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EZ_Inventory
{
    public class Product
    {

        public int  ID { get; set; }
        public int UPC { get; set; }
        public string Name { get; set; }
        public string Vendor { get; set; }
        public bool IsActive { get; set; }
        public double UnitCost { get; set; }
        public double RetailPrice { get; set; }
        public int UnitsInStock { get; set; }

    }
}
